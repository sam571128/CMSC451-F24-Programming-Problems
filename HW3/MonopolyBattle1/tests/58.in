31 961 1000000000
18 10
19 23
12 20
3 6
1 26
14 15
30 2
16 22
21 6
10 9
24 15
31 18
4 21
21 13
6 30
31 12
15 11
11 3
22 2
27 17
22 23
10 8
28 3
3 13
17 21
5 2
2 1
1 31
22 1
22 24
18 19
15 17
30 22
1 10
30 23
1 1
2 20
7 18
20 13
16 7
10 12
12 13
13 14
2 31
2 29
27 14
6 14
22 16
24 24
13 19
25 9
6 19
30 7
19 1
31 5
9 27
2 6
2 5
29 26
29 15
18 6
19 22
28 1
12 29
12 10
2 25
21 1
5 22
19 11
1 11
17 9
3 2
26 25
5 26
14 31
20 21
25 29
9 2
25 19
26 14
20 7
8 2
27 23
23 6
11 21
4 2
2 23
10 20
28 23
25 8
6 12
14 28
9 29
13 15
23 9
6 28
21 20
28 9
24 14
10 2
6 16
3 12
27 3
12 18
19 24
17 15
12 15
31 15
19 20
2 3
16 30
25 3
9 16
5 19
5 11
22 25
19 19
18 11
12 5
12 22
15 20
14 20
21 9
28 31
26 11
13 7
16 29
30 1
1 12
19 5
3 16
26 31
28 13
14 25
30 18
1 30
29 3
11 6
5 20
31 7
17 11
9 26
17 1
25 12
10 19
7 29
25 5
17 18
25 24
31 19
3 20
9 17
31 30
14 22
31 23
2 15
13 12
13 21
5 28
16 11
7 21
23 30
29 23
11 24
11 4
3 11
26 2
22 7
31 4
22 3
13 10
7 28
23 18
16 23
27 31
18 2
29 25
18 16
21 21
5 31
11 11
7 25
1 3
25 16
31 1
14 6
2 18
14 19
21 30
8 22
15 24
28 5
31 27
8 13
5 13
4 4
4 27
1 19
29 12
21 26
30 8
26 21
20 1
6 24
14 12
10 14
5 29
26 1
1 23
29 14
16 31
10 5
24 11
11 22
14 9
15 27
27 20
17 23
27 7
16 26
4 31
31 14
24 1
18 9
25 6
10 4
11 18
4 9
16 14
9 24
16 9
10 21
10 7
12 4
13 24
11 7
21 27
7 22
3 22
12 2
11 9
20 11
2 30
19 25
17 31
14 7
1 28
28 10
24 6
7 20
23 25
8 1
13 6
9 31
6 5
3 26
12 28
19 27
1 14
10 17
18 31
17 3
28 28
14 11
12 27
28 27
4 14
6 31
22 12
4 8
1 2
6 7
7 1
5 6
12 23
2 13
5 25
9 15
23 27
31 8
8 11
8 9
8 27
17 14
25 17
30 12
12 26
6 29
15 10
24 17
5 12
27 9
4 24
27 10
16 3
17 10
24 18
14 8
30 11
20 15
15 16
26 15
24 12
27 12
15 5
15 7
25 2
7 2
3 25
12 25
8 4
15 9
27 11
6 27
25 28
1 29
11 17
9 21
24 31
4 23
20 16
27 1
20 3
23 22
26 30
2 10
24 25
7 31
8 5
12 9
29 22
13 9
20 30
18 22
8 20
9 7
15 15
18 4
20 28
3 4
24 20
2 22
20 18
7 16
31 17
15 3
4 10
30 3
14 18
17 4
6 25
4 19
1 27
3 9
23 2
31 21
27 21
28 20
30 27
20 2
5 23
9 23
23 31
12 19
5 17
22 29
26 24
27 24
21 11
13 26
2 7
21 19
6 11
23 7
25 1
25 4
27 6
4 25
18 14
18 29
22 5
16 10
20 5
7 3
8 29
2 28
14 14
24 29
20 10
3 15
31 26
20 17
15 4
22 15
3 23
26 20
2 16
25 18
22 18
7 10
23 20
30 16
27 29
24 26
22 19
10 31
21 5
19 2
14 16
11 20
4 26
22 30
23 4
1 7
13 11
18 20
23 24
7 19
4 30
5 21
13 2
17 25
15 19
10 22
23 21
20 12
24 27
10 16
13 16
26 22
8 31
19 12
19 7
25 7
5 9
16 16
7 15
31 16
11 19
17 5
22 20
10 24
1 8
29 1
27 25
29 8
5 10
26 28
23 13
15 13
20 26
23 28
29 7
19 3
3 28
22 8
23 15
27 2
30 19
19 18
25 13
25 26
15 18
20 29
8 15
13 30
26 10
26 3
16 25
20 8
29 16
13 27
1 17
13 28
4 6
28 2
20 23
7 9
6 1
20 9
14 29
29 11
23 23
21 8
16 27
29 4
16 28
8 21
17 30
7 27
28 19
17 19
14 2
10 15
29 29
18 24
17 22
1 13
11 12
12 14
9 10
11 31
20 27
12 16
26 12
27 30
29 27
27 28
30 21
11 25
18 27
8 17
14 21
2 9
26 23
2 19
27 8
14 24
28 26
1 25
18 3
29 2
30 25
27 13
28 21
2 14
18 1
24 4
15 30
18 5
16 21
19 26
28 30
18 7
13 13
6 13
16 4
5 18
17 27
10 6
21 22
15 25
25 10
20 20
18 21
20 22
9 30
3 30
13 8
28 4
30 26
19 4
9 3
3 1
2 26
27 5
4 20
7 24
24 2
22 11
18 23
21 25
18 25
14 3
7 23
3 21
14 10
9 1
24 28
20 4
22 26
4 5
22 10
19 13
6 18
27 18
21 10
8 26
23 29
7 30
15 23
28 22
13 18
30 29
3 8
13 31
12 1
22 6
19 29
20 31
31 13
21 16
1 16
9 20
8 16
5 30
19 14
10 29
9 22
10 13
28 8
12 17
3 19
1 18
17 2
19 21
10 1
5 3
20 14
9 19
19 17
19 30
24 23
8 23
4 3
4 13
23 17
9 13
19 6
28 11
21 12
10 23
10 26
28 17
17 7
5 7
27 27
13 17
9 28
21 28
9 5
31 9
6 10
28 25
27 15
7 4
2 2
25 30
19 15
31 25
26 9
11 1
9 11
7 12
18 12
3 7
28 24
20 25
16 6
26 6
31 6
12 8
30 30
29 30
2 24
6 21
8 19
5 8
17 13
4 7
18 26
9 9
26 5
14 23
13 22
30 31
5 27
21 18
21 15
29 19
9 25
16 15
25 23
13 1
15 31
10 25
31 24
5 4
26 13
9 12
29 21
23 3
13 3
7 7
28 14
16 18
18 13
18 15
3 31
30 15
22 21
31 22
11 23
15 28
24 30
24 10
23 26
5 1
6 23
15 8
11 2
3 5
8 3
15 26
31 3
25 27
28 6
6 8
1 15
3 17
10 3
6 22
28 16
26 29
15 29
2 12
4 22
31 10
7 17
25 11
30 28
17 17
19 10
22 22
3 18
1 21
10 28
12 21
7 13
17 20
16 2
20 6
19 8
4 16
14 4
3 27
21 29
5 16
8 6
16 24
23 16
6 20
16 5
7 8
14 1
24 5
30 6
29 6
19 9
8 8
13 29
31 29
12 3
22 9
29 24
14 13
29 10
22 4
29 17
30 5
21 23
27 19
1 9
24 16
20 24
22 27
27 4
1 22
10 11
17 16
12 12
24 19
30 10
13 25
26 26
26 7
21 2
28 18
8 10
5 15
23 1
11 28
23 19
17 24
15 6
24 13
12 24
30 9
21 7
8 25
31 11
7 11
1 5
3 10
30 24
25 14
1 20
6 3
17 12
14 5
26 18
2 27
9 18
12 31
11 13
19 16
26 8
11 10
18 28
8 24
6 17
6 15
25 31
1 4
11 26
16 8
24 22
11 29
27 26
4 1
11 14
18 17
15 1
16 1
4 17
11 16
10 10
31 28
11 8
8 28
3 14
5 24
21 17
30 4
2 11
4 28
11 27
30 17
24 7
16 13
12 30
22 17
28 12
29 5
21 24
26 4
16 20
7 14
29 31
13 23
9 4
24 8
28 15
10 18
31 20
18 18
21 14
14 27
14 26
26 27
8 7
6 2
26 16
15 21
14 30
18 30
25 15
16 12
19 31
10 27
16 19
29 13
28 29
24 3
7 26
25 20
26 17
3 29
23 10
22 31
22 28
23 14
25 22
20 19
13 5
14 17
15 12
1 6
26 19
21 31
8 30
29 9
9 14
31 2
29 28
1 24
6 26
7 6
11 15
30 20
2 4
9 6
17 26
3 3
23 8
8 18
19 28
21 3
6 4
25 25
12 11
4 11
9 8
6 6
4 29
16 17
2 21
2 8
8 12
11 30
12 6
15 2
15 22
5 5
4 18
11 5
29 20
24 21
24 9
15 14
22 14
5 14
21 4
17 6
3 24
13 4
17 29
25 21
4 12
30 14
8 14
7 5
27 22
12 7
27 16
28 7
10 30
23 5
22 13
23 11
29 18
4 15
2 17
23 12
18 8
13 20
31 31
30 13
6 9
17 8
17 28
