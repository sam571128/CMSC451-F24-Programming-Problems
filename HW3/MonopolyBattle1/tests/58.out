36 43 43 165 55 43 48 36 36 36 43 36 39 43 43 48 43 36 36 36 43 43 36 43 66 36 56 92 45 43 36 